.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+- [ ] I have the right to license submitted text in this Pull Request under the mentioned [license](LICENSE) and hereby license everything in this Pull Request under it.
+      If this text is from a third party, I have the appropiate right to license it under the mentioned [license](LICENSE).
```

Comment:
```suggestion
- [ ] I have the right to license submitted content in this Pull Request under the mentioned [license](LICENSE) and hereby license everything in this Pull Request under it.
      If this content is from a third party, I have the appropiate right to license it under the mentioned [license](LICENSE).
```

This might be a bit more exhaustive than "text".

Reply:
> This might be a bit more exhaustive than "text".

Maybe we should use `Work` as the word, as this seems to be the word used in the license. Maybe even something like `content (Work)` or so. What do you think?

Reply:
i like content since work implies that one has done it themselves so content is more exhaustive

Reply:
I think "content (work)" is fine

Reply:
I changed it to `content (Work)`, `Work` is written intentionally with a capital `W`, for more information look into the license.